zds/forum/models.py

```diff
@@ -130,7 +130,9 @@ def get_last_message(self):
         :return: the last message on the forum, if there are any.
         """
         try:
-            return Post.objects.filter(topic__forum=self).order_by("-pubdate").all()[0]
+            last_post = Post.objects.select_related("topic").filter(topic__forum=self).order_by("-pubdate").all()[0]
```

Comment:
Tu n'y a pas touché donc tu n'as pas d'obligation à modifier cela mais j'imagine que ça doit simplifier la requête SQL donc vu que c'est appelé plein de fois ce serait chouette de modifier ça !
```suggestion
            last_post = Post.objects.select_related("topic").filter(topic__forum=self).order_by("-pubdate").first()
```